helm/templates/routing/manager.networkpolicy.yaml

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if ne .Values.cluster.kind "Openshift" }}
```

Comment:
```suggestion
```

I'd also apply the NetworkPolicies in OpenShift. This would align them with the general NetworkPolicies and we can change them via PRs.

Reply:
Instead, I'd only add the whole section if `ingressNamespace` is set. Otherwise, it would be breaking change and has to be marked accordingly.